sqlstore/src/main/resources/db/migration_afterMigrate/10_pinery_queryInstruments.sql

```diff
@@ -0,0 +1,15 @@
+--StartNoTest
+DELIMITER //
+
+DROP PROCEDURE IF EXISTS queryAllInstruments//
+CREATE PROCEDURE queryAllInstruments() BEGIN
+  PREPARE stmt FROM 'SELECT sr.referenceId
```

Comment:
Prepared statements work to prevent SQL injection, but is there any purpose to using them within a stored procedure for queries that have no parameters and are only executed once (per procedure call)?

Reply:
It's intended as a developer-friendliness thing -- using prepared statements for both the parameterized as well as paramater-less queries means there are fewer lines in a diff of the two files.